### PR TITLE
Gl 649 add Exemptions questions pages Cat 1 & 2

### PR DESCRIPTION
**Pull request by alexdesi**
### Jira link

https://transformuk.atlassian.net/browse/GL-649
https://transformuk.atlassian.net/browse/GL-650

### What?
Create the question exemptions page Cat1 and Cat 2
### Why?

I am doing this because:
To assess the correct category for the user.

### CODE REVIEW

**Comment by alexdesi** [comment]
Screenshot:
exemptions_questions_page_cat1

**Comment by alexdesi** [comment]
Screenshot Q1
exemptions_questions_page_cat1

File: spec/requests/green_lanes/moving_requirements_controller_spec.rb [hunk 1]
```diff
@@ -74,7 +71,43 @@
...
+  end
+
+  describe 'GET #cat_1_exemptions_questions_edit' do
+    let(:make_request) { get cat_1_questions_green_lanes_check_moving_requirements_path }
+
+    it { is_expected.to have_http_status(:ok) }
+    it { is_expected.to render_template('cat_1_exemptions_questions_edit') }
+  end
+
+  describe 'PUT #cat_1_exemptions_questions_update' do
```
Comment (shjohnson) [inline spec/requests/green_lanes/moving_requirements_controller_spec.rb]: delete this

File: spec/requests/green_lanes/moving_requirements_controller_spec.rb [hunk 2]
```diff
@@ -74,7 +71,43 @@
...
+  end
+
+  describe 'GET #cat_2_exemptions_questions_edit' do
+    let(:make_request) { get cat_2_questions_green_lanes_check_moving_requirements_path }
+
+    it { is_expected.to have_http_status(:ok) }
+    it { is_expected.to render_template('cat_2_exemptions_questions_edit') }
+  end
+
+  describe 'PUT #cat_2_exemptions_questions_update' do
```
Comment (shjohnson) [inline spec/requests/green_lanes/moving_requirements_controller_spec.rb]: delete this

File: spec/requests/green_lanes/moving_requirements_controller_spec.rb [hunk 3]
```diff
@@ -74,7 +71,43 @@
...
+      expect(make_request).to redirect_to('some_path')
+      expect(flash[:success]).to eq 'Exemptions updated successfully.'
+    end
+  end
+
+  describe 'GET #cat_2_exemptions_questions_edit' do
+    let(:make_request) { get cat_2_questions_green_lanes_check_moving_requirements_path }
+
+    it { is_expected.to have_http_status(:ok) }
+    it { is_expected.to render_template('cat_2_exemptions_questions_edit') }
```
Comment (shjohnson) [inline spec/requests/green_lanes/moving_requirements_controller_spec.rb:89]: this can be changed

**Review by shjohnson — APPROVED** [review]
LGTM

**Pull request merged**